<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QFrame" name="gridFrame">
             <widget class="QComboBox" name="comboBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>131</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="gridFrame1">
             <widget class="QComboBox" name="comboBox_2">
              <property name="geometry">
               <rect>
                <x>9</x>
                <y>9</y>
                <width>121</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="gridFrame">
             <widget class="QComboBox" name="comboBox_3">
              <property name="geometry">
               <rect>
                <x>9</x>
                <y>9</y>
                <width>111</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QWidget" name="">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit_3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1391384</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="sizeIncrement">
                 <size>
                  <width>10</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
                <property name="maximumBlockCount">
                 <number>10</number>
                </property>
                <property name="backgroundVisible">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit"/>
              </item>
              <item>
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Read</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QTableWidget" name="tableWidget">
                <property name="acceptDrops">
                 <bool>true</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <italic>true</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Select Column for X &amp; Y axis</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox"/>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_2"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>plot</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_4">
                  <property name="text">
                   <string>clear</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuCharts">
    <property name="title">
     <string>Charts</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionBar_Chart"/>
    <addaction name="actionPie_Chart"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuCharts"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="actionBar_Chart"/>
   <addaction name="actionPie_Chart"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>:/img/newtab.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/img/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/img/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset>
     <normalon>:/img/linechart.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Line Chart</string>
   </property>
  </action>
  <action name="actionBar_Chart">
   <property name="icon">
    <iconset>
     <normalon>:/img/barchart.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Bar Chart</string>
   </property>
  </action>
  <action name="actionPie_Chart">
   <property name="icon">
    <iconset>
     <normalon>:/img/piechart.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pie Chart</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
